README.md

```diff
@@ -58,3 +58,188 @@ select * where {
     <urn:a> ?p ?o  .
 }
 ```
+
+# Secured Kafka Sink with FailedRecordProducer Configuration
```

Comment:
`FailedRecordProducer` is a class that mean something to us. I'd rename this to 

# Enable Security for Failed Messaged sent to DLQ

or something similar.